openmetadata-ui/src/main/resources/ui/playwright/e2e/auth.teardown.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { test as teardown } from '@playwright/test';
+import { AdminClass } from '../support/user/AdminClass';
+import { resetPolicyChanges } from '../utils/authTeardown';
+
+teardown('restore the organization roles and policies', async ({ page }) => {
+  const admin = new AdminClass();
+
+  // Reset the default organization roles and policies
+  await resetPolicyChanges(page, admin);
+});
```

Comment:
it will run after each test? just wanted to know whats the idea behind it?

Reply:
Just like setup, it run only once when all the tests are finished running